frontend/i18n/pl/errors.js

```diff
@@ -0,0 +1,7 @@
+export default {
+  fileCannotUpload:
+  'Plik(i) nie mogą zostać przesłane. Być może nieprawidłowy format. Sprawdź dostępne formaty i następujące pliki: ',
+  labelCannotCreate:
+  'Nie można utworzyć etykiety. Nie możesz użyć tej samej nazwy etykiety lub klawisza skrótu.',
+invalidUserOrPass: 'Nieprawidłowa nazwa użytkownika lub hasło albo coś poszło nie tak.'
+}
```

Comment:
I think this file is accidentally duplicated, because the correct one is inside the `projects` folder (i.e. `frontend/i18n/pl/projects/errors.js`). Maybe this file should be deleted 😅